dev-python/pygobject/pygobject-3.52.3.ebuild

```
@@ -0,0 +1,95 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_EXT=1
+DISTUTILS_USE_PEP517=no
+GNOME_TARBALL_SUFFIX="gz"
+PYTHON_COMPAT=( python3_{10..13} pypy3 pypy3_11 )
+
+inherit gnome.org meson virtualx xdg distutils-r1
+
+DESCRIPTION="Python bindings for GObject Introspection"
+HOMEPAGE="
+	https://pygobject.gnome.org/
+	https://gitlab.gnome.org/GNOME/pygobject/
+"
+SRC_URI+="
+	https://github.com/python/pythoncapi-compat/archive/0a8b2c56331a31d7f7096faaa1c1c26467b51c15.tar.gz -> \
+		${P}_custom_pythoncapi-compat.gh.tar.gz
```

```suggestion
	https://github.com/python/pythoncapi-compat/archive/${COMMIT}.tar.gz -> \
		${P}_${COMMIT}_pythoncapi-compat.gh.tar.gz
```

Let's put the hash in some variable since we are using it multiple times, this will make it easier to bump to a newer commit later if we want to. Let's also put the hash in the tarball name for the same reason.